core/server/data/migrations/versions/4.43/2022-04-01-10-13-add-post-newsletter-relation.js

```diff
@@ -48,14 +62,19 @@ module.exports = {
 
         logging.info(`Removing ${table}.${column} column`);
 
-        if (DatabaseInfo.isSQLite(knex)) {
-            await commands.dropColumn(table, column, knex, columnDefinition);
-            return;
-        }
+        await knex.schema.alterTable(table, function (t) {
```

Comment:
Can we use the `dropForeign`[^1] command instead? That handles checking for the existence of the constraint and also handles SQLite cases 

[^1]: https://github.com/TryGhost/Ghost/blob/main/core/server/data/schema/commands.js#L220